# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1489608597 14400
#      Wed Mar 15 16:09:57 2017 -0400
# Node ID 9a0e33470da57e85b04282e3b69df013da08b91c
# Parent  67d741321e215bd15dfa908a3475e7be4bb44b9f
use "run-time condition" consistently in previous change

* __run_test_suite__.m, test.m: Spell run-time consistently.

diff --git a/scripts/testfun/__run_test_suite__.m b/scripts/testfun/__run_test_suite__.m
--- a/scripts/testfun/__run_test_suite__.m
+++ b/scripts/testfun/__run_test_suite__.m
@@ -69,26 +69,26 @@ function [pass, fail, xfail, skip, rtski
         dp += p;
         dn += n;
         dxf += xf;
         dsk += sk;
         drtsk += rtsk;
       endfor
       puts ("\nSummary:\n\n");
       nfail = dn - dp - dxf;
-      printf ("  PASS                   %6d\n", dp);
-      printf ("  FAIL                   %6d\n", nfail);
+      printf ("  PASS                         %6d\n", dp);
+      printf ("  FAIL                         %6d\n", nfail);
       if (dxf > 0)
-        printf ("  XFAIL                  %6d\n", dxf);
+        printf ("  XFAIL                        %6d\n", dxf);
       endif
       if (dsk > 0)
-        printf ("  SKIPPED (feature)      %6d\n", dsk);
+        printf ("  SKIPPED (feature)            %6d\n", dsk);
       endif
       if (drtsk > 0)
-        printf ("  SKIPPED (runtime test) %6d\n", drtsk);
+        printf ("  SKIPPED (run-time condition) %6d\n", drtsk);
       endif
       puts ("\n");
       printf ("See the file %s for additional details.\n", logfile);
       if (dxf > 0)
         puts ("\n");
         puts ("Items listed as XFAIL above are known bugs.\n");
         puts ("Bug report numbers for them may be found in the log file:\n");
         puts (logfile);
@@ -145,20 +145,20 @@ function print_pass_fail (p, n, xf, sk, 
 
   if ((n + sk + rtsk) > 0)
     printf (" PASS   %4d/%-4d", p, n);
     nfail = n - p - xf;
     if (nfail > 0)
       printf ("\n%71s %3d", "FAIL ", nfail);
     endif
     if (sk > 0)
-      printf ("\n%71s %3d", "(feature) SKIP ", sk);
+      printf ("\n%71s %3d", "(missing feature) SKIP ", sk);
     endif
     if (rtsk > 0)
-      printf ("\n%71s %3d", "(runtime test) SKIP ", rtsk);
+      printf ("\n%71s %3d", "(run-time condition) SKIP ", rtsk);
     endif
     if (xf > 0)
       printf ("\n%71s %3d", "XFAIL", xf);
     endif
   endif
   puts ("\n");
 
 endfunction
diff --git a/scripts/testfun/test.m b/scripts/testfun/test.m
--- a/scripts/testfun/test.m
+++ b/scripts/testfun/test.m
@@ -78,17 +78,17 @@
 ## @var{fid}.
 ##
 ## When called with just a single output argument @var{success}, @code{test}
 ## returns true if all of the tests were successful.  If called with more
 ## than one output argument then the number of successful tests (@var{n}),
 ## the total number of tests in the file (@var{nmax}), the number of xtest
 ## failures (@var{nxfail}), the number of tests skipped due to missing
 ## features (@var{nskip}), and the number of tests skipped due to
-## runtime conditions (@var{nrtskip}) are returned.
+## run-time conditions (@var{nrtskip}) are returned.
 ##
 ## Example
 ##
 ## @example
 ## @group
 ## test sind
 ## @result{}
 ## PASSES 5 out of 5 tests
@@ -709,17 +709,17 @@ function [__n, __nmax, __nxfail, __nskip
         printf ("PASSES %d out of %d test%s\n", __successes, __tests,
                ifelse (__tests > 1, "s", ""));
       endif
       if (__xskip)
         printf ("Skipped %d test%s due to missing features\n", __xskip,
                 ifelse (__xskip > 1, "s", ""));
       endif
       if (__xrtskip)
-        printf ("Skipped %d test%s due to runtime tests\n", __xrtskip,
+        printf ("Skipped %d test%s due to run-time conditions\n", __xrtskip,
                 ifelse (__xrtskip > 1, "s", ""));
       endif
     else
       printf ("%s%s has no tests available\n", __signal_empty, __file);
     endif
   elseif (__grabdemo)
     __n    = __demo_code;
     __nmax = __demo_idx;
